pom.xml

```diff
@@ -9,7 +9,7 @@
     <relativePath />
   </parent>
   <artifactId>applitools-eyes</artifactId>
-  <version>1.14-SNAPSHOT</version>
+  <version>${revision}</version>
```

Comment:
```suggestion
  <version>${changelist}</version>
```

Reply:
As I wrote, I'd prefer to use the revision and not the changelist, since I understand it better, and the changelist is not so clear to me. But I'm new to this so perhaps you could explain: I want the next version published to be 1.14. If I'll need to update again then 1.15 and so on until a major change is required.
Can I do that with changelist?

Reply:
that's not really supported, cc @jglick / @NotMyFault if you want to weight in.

> It is not recommended to implement actual semantic versioning with automated releases performed by CD, as that requires great care in always changing the revision as part of the changes that semantically would require a revision change for the next release. Otherwise, automated releases may have version numbers that semantically would not make sense.

Reply:
> I want the next version published to be 1.14. If I'll need to update again then 1.15 and so on until a major change is required.

The maven release plugin may be the better alternative then. It allows you to control the versions you want to publish, and when you publish it.
Releases via CD are not made to follow a semver-like or major.minor pattern, and use a different versioning scheme.
Automatic releases based on the maven release plugin don't exist.

Reply:
> I don't know it so an example would be nice

See https://www.jenkins.io/doc/developer/publishing/releasing-manually/

> meaning I'll still need the secrets. If I switch to it, will they still appear in [the secrets page](https://github.com/jenkinsci/applitools-eyes-plugin/settings/secrets/)?

No, you don't. These files are setup locally on your machine, not GitHub.

Reply:
but I'll need to publish them to the same place where the CD publishes, won't I?

Reply:
That's what the maven release plugin does, once you set it up on your machine.

Reply:
In the docs I see I need to log into Artifactory using Username and Password to get the settings. Those credentials are mensioned as "your Jenkins community account":

> Prerequisite: You need to have logged in once to [Artifactory](https://repo.jenkins-ci.org/) and [Jira](https://issues.jenkins.io/) with your Jenkins community account (this is the same as the account you would use to login to Jira) before you can be added to a permissions target.

Where or how can I get those credentials if I don't have them?

Reply:
They are your credentials for https://accounts.jenkins.io/

Reply:
> Last time the password was updated was in 2020, and it doesn't work now.

If you need help with your accounts.jenkins.io account, file a [helpdesk](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=triage%2Caccounts&template=2-account-issue.yml) ticket.

Reply:
Account issue resolved in https://github.com/jenkins-infra/helpdesk/issues/3564